### PR TITLE
[Feature] Update employee endpoint

### PR DESCRIPTION
**Pull request by chrischan325**
## Notion ticket link

Update employee endpoint (https://www.notion.so/uwblueprintexecs/Update-employee-endpoint-1f94d548abc545a8b88ad80c647426fc)

## Implementation description
* added updateEmployee resolver 
* created schema for the API
* added authorization logic so that only admins can access the endpoint

## Notes
* 

## Checklist
- [x] My PR name is descriptive, is in imperative tense and starts with one of the following: `[Feature]`,`[Improvement]` or `[Fix]`,
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the RCD team on GitHub, or specific people who are associated with this ticket

### CODE REVIEW

File: lib/employees/schema.ts [hunk 1]
```diff
@@ -21,4 +22,17 @@ export default `
   type CreateEmployeeResult {
     ok: Boolean!
   }
+
+  input UpdateEmployeeInput {
+    id: ID!
+    firstName: String!
+    lastName: String!
+    email: String
```
Comment (OustanDing) [inline lib/employees/schema.ts:30]:
```suggestion
    email: String!
```
We want this field to be mandatory, as per the Edit Admin modal design

**Review by OustanDing — CHANGES_REQUESTED** [review]
Just had one small comment. Everything else looks great!

**Comment by OustanDing** [comment]
Also please change the PR title to match the format of the other tickets :)

File: lib/employees/schema.ts [hunk 2]
```diff
@@ -21,4 +22,17 @@ export default `
...
     ok: Boolean!
   }
+
+  input UpdateEmployeeInput {
+    id: ID!
+    firstName: String!
+    lastName: String!
+    email: String!
+    role: Role!
+    active: Boolean
```
Comment (OustanDing) [inline lib/employees/schema.ts:32]:
```suggestion
```
It should be our `DELETE` endpoint that handles the `active` flag, so we don't want to allow it to be changed in the `UPDATE` endpoint

**Review by OustanDing — CHANGES_REQUESTED** [review]
One more thing. Also, don't forget to regenerate types

File: lib/employees/schema.ts (hunk 2 above)
Comment (chrischan325) [reply lib/employees/schema.ts:32]: I see I see

**Review by OustanDing — APPROVED** [review]
LGTM!

**Pull request merged**